js/fetch.js

```diff
@@ -659,85 +659,85 @@ define(["jquery"], function($) {
             id: 109,
             word: 'Pakistan',
             category: 'country',
-            hint: '',
+            hint: 'A country in South Asia. It is the world\'s fifth-most populous country, and has the world\'s second-largest Muslim population.',
         },
         {
             id: 110,
             word: 'India',
             category: 'country',
-            hint: '',
+            hint: 'A country in South Asia. It is the seventh-largest country by area, the second-most populous country, and the most populous democracy in the world.',
         },
         {
             id: 111,
             word: 'Bangladesh',
             category: 'country',
-            hint: '',
+            hint: 'A country in South Asia. It is the eighth-most populous country in the world.',
         },
         {
             id: 112,
             word: 'Sri Lanka',
             category: 'country',
-            hint: '',
+            hint: 'Formerly known as Ceylon. An island country in South Asia.',
         },
         {
             id: 113,
             word: 'Afghanistan',
             category: 'country',
-            hint: '',
+            hint: 'A landlocked country located at the crossroads of Central and South Asia. Referred to as the Heart of Asia.',
         },
         {
             id: 114,
             word: 'Iraq',
             category: 'country',
-            hint: '',
+            hint: 'A country in Western Asia. The capital and largest city is Baghdad.',
```

Comment:
@mhamid49 I think writing capital would be very obvious. The hint should be a hidden message.

Reply:
@ammara-sadaf I think it's fine as of now. We have to make it dynamic anyways.